index.html

```diff
@@ -1,88 +1,320 @@
-                          </li>
-
-                          <li>
-                            
-                                  <i class="fa-brands fa-m"></i>
-                              
-                          </li>
-
-                          <li>
-                              
-                                  <i class="fa-brands fa-github"></i>
-                              
-                          </li>
-                        
-                          <li>
-      
-                                  <i class="fa-solid fa-hand-peace"></i>
-                              
-                          </li>
-                      </ul>
-                  </div>
-              </div>
+    <div class="background">
+      <div class="wrapper">
+        <div class="intro-text">
+          <h1>
+            I'm CarenSiya
+            <br>
+            Glad to see you!
+          </h1>
+          <p></p>
+        </div>
+        <div class="main-text">
+          <p class="text-description">
+            I’m a software developer! I can help you build a product , feature or website Look through some of my work
+            and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+          </p>
+          <div class="social">
+            <h2 class="connection">LET'S CONNECT</h2>
+            <ul class="media-icons">
+              <li>
+
+                <i class="fa-brands fa-twitter"></i>
+
+              </li>
+
+              <li>
+
+                <i class="fa-brands fa-linkedin"></i>
+
+              </li>
+
+              <li>
+
+                <i class="fa-brands fa-m"></i>
+
+              </li>
+
+              <li>
+
+                <i class="fa-brands fa-github"></i>
+
+              </li>
+
+              <li>
+
+                <i class="fa-solid fa-hand-peace"></i>
+
+              </li>
+            </ul>
           </div>
+        </div>
       </div>
+    </div>
   </section>
 
+  <section class="container">
+    <div class="card">
+      <img src="/images/image-1.png" alt="" class="image">
+      <div class="left-card1-block">
+        <h1 class="project-title">Tonic</h1>
+        <!-- Flex -->
+        <ul class="project-badges">
+          <li class="text-1">Canopy</li>
+          <img src="/images/counter-images.png" alt="" class="counter-image">
+          <li class="text-2">
+            <p>Back&nbspEnd&nbspDev</p>
+          </li>
+          <img src="/images/counter-images.png" alt="" class="counter-image">
+          <li class="text-3">2015</li>
+        </ul>
+
+        <p class="project-description">
+          A daily selection of privately personalized reads; no accounts or sign-ups required.
+        </p>
+        <br>
+        <ul class="language-categories">
+          <li>html</li>
+          <li>css</li>
+          <li>javascript</li>
+        </ul>
+        <button class="submit-btn" type="submit">See Project</button>
```

Comment:
- Let's also add the missing interaction between buttons by indicating when it's active, hovered, pressed, etc. kindly check the Figma design to see the correct color used to improve our design.